gallery/src/styles/NxCard/NxCardPageExamples.scss

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
This file needs to be included as a code example

Reply:
> This file needs to be included as a code example

https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/3b7359d1d9654d0acb9a78749a333b4892173239